pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md

```diff
@@ -0,0 +1,298 @@
+*   Initially hide the email content until the AMP library is loaded by placing the AMP for Email boilerplate in the `<head>`.
+
+```html
+<head>
+...
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+</head>
+```
+
+
+
+
+## AMP Specific Tag Replacements 
+
+Since the AMP for Email library is a subset of the AMP HTML library, many of the same rules apply; AMP specific tags replace resource heavy HTML tags and require a defined width and height. This allows the AMP boilerplate to hide content until it has an idea of how it looks on the user's device. 
+
+
+### Images
+
+To paint the page effectively, all `<img>` tags are replaced with `[<amp-img>](https://www.ampproject.org/docs/reference/components/amp-img)`. The `<amp-img>` tag requires a defined width and height and supports [AMP's layout system](https://www.ampproject.org/docs/design/amp-html-layout).
+
+
+```
+<amp-img src="https://link/to/img.jpg" 
+    width="100" 
+    height="100" 
+    layout="responsive">
+</amp-img>
+```
+
+The `<amp-img>` tag comes with powerful, built-in ways to control responsive design and set fallbacks.
+
+[tip type="note"]
+    Read more about using the AMP [layout and media queries](https://www.ampproject.org/docs/design/responsive/control_layout) and how to set [image fallbacks](https://www.ampproject.org/docs/design/responsive/placeholders). 
+[/tip]
+
+### GIFs
+
+AMP has created `[<amp-anim>](https://www.ampproject.org/docs/reference/components/amp-anim)`, a specific tag for GIF images that allows the AMP runtime to reduce CPU usage when the animation is off-screen. Similar to `<amp-img>` the width and height is defined and the element must include a closing tag.
+
+
+```
+<amp-anim 
+    width="400" 
+    height="300" 
+    src="my-gif.gif">
+</amp-anim>
+```
+
+
+Additionally, it supports an optional `placeholder` child to display while the `src` file is loading, and supports the AMP layout system. 
+
+
+```
+<amp-anim width=400 height=300 src="my-gif.gif" layout="responsive">
+  <amp-img placeholder width=400 height=300 src="my-gif-screencap.jpg">
+  </amp-img>
+</amp-anim>
+```
+
+
+
+# Emails, with style 
+
+Like all email clients, AMP allows for inline `style` attributes, but also supports CSS within the `<style amp-custom>` tag inside the head of the email.
+
+
+```html
+...
+<style amp-custom>
+  /* any custom styles go here. */
+  body {
+    background-color: white;
+  }
+  amp-img {
+    border: 5px solid black;
+  }
+</style>
+...
+</head>
+```
+[tip type="important"]
+    AMP enforces a size limit of 50,000 bytes for styling.
+[/tip]    
+
+In Gmail, AMP email supports attributes and pseudo-classes disallowed in other email MIME types. 
```

Comment:
I'm not sure if we should include something Gmail-specific here, but I'm fine with keeping it until we find it a better home.

Reply:
Also, I think we should mention that the AMP validator, for most part, won't catch CSS attributes / pseudo-classes that are not allowed. Instead, the email client just strips or ignores these.